commit ca41a29b8bb19f0240eac0f872a7b958fb1f5f19
Author: Patrick Steinhardt <ps@pks.im>
Date:   Thu Sep 5 08:04:37 2024 +0200

    clar: stop passing a timezone to gettimeofday(3P)
    
    According to gettimeofday(3P), passing a non-NULL timezone pointer to
    the function is unspecified behaviour. This is also being warned about
    by compilers when compiling with strict C90 standard and without most of
    the extensions.
    
    Adapt the code accordingly.

diff --git t/unit-tests/clar.c t/unit-tests/clar.c
index cef0f02..e593bb1 100644
--- t/unit-tests/clar/clar.c
+++ t/unit-tests/clar/clar.c
@@ -271,9 +271,7 @@ static double clar_time_diff(clar_time *start, clar_time *end)
 
 static void clar_time_now(clar_time *out)
 {
-	struct timezone tz;
-
-	gettimeofday(out, &tz);
+	gettimeofday(out, NULL);
 }
 
 static double clar_time_diff(clar_time *start, clar_time *end)
